pkg/domain/infra/abi/containers.go

```diff
@@ -1188,13 +1188,15 @@ func (ic *ContainerEngine) ContainerRun(ctx context.Context, opts entities.Conta
 	return &report, nil
 }
 
-func (ic *ContainerEngine) GetContainerExitCode(ctx context.Context, ctr *libpod.Container) int {
+func (ic *ContainerEngine) GetContainerExitCode(ctx context.Context, ctr *libpod.Container) *int {
```

Comment:
Why is this function changed?

Reply:
Changing that as I mistook it for the same function as libpod>stage